benches/misc/coprocessor/codec/batch/mod.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use test::Bencher;
+use tikv::coprocessor::codec::batch::*;
+use tikv::coprocessor::codec::mysql::*;
+
+#[bench]
+fn bench_column_agg_long(b: &mut Bencher) {
+    let rows = 1024;
+    let mut long_col = VectorValue::with_capacity(rows, cop_datatype::EvalType::Int);
+    for row_id in 0..rows {
+        if row_id & 1 == 0 {
+            long_col.push_int(None);
+            continue;
+        }
+        long_col.push_int(Some(row_id as i64));
```

Comment:
Currently VectorValue does not support adding values. `push_int` is only for tests because it always checks type (but we can check type only once). Maybe comment it out for now?